api/controllers/service_api/app/message.py

```diff
@@ -109,8 +109,47 @@ def get(self, app_model: App):
         return {"data": feedbacks}
 
 
+class MessageDetailApi(Resource):
+    message_detail_fields = {
```

Comment:
Can we use https://github.com/langgenius/dify/blob/8537abfff808ba0647cb4e266b5d2b018eb26528/api/fields/conversation_fields.py#L62 here?